# Designation of 5 initial B.1 sublineages

#### Cornelius Roemer, Emma B. Hodcroft, Christian Happi, Crystal Gigante, Ifedayo Adetifa, Placide Mbala, Richard Njouom, Emmanuel Nakoune, Anise Happi, Naemeka Ndodo, Oyeronke Ayansola, Gerald Mboowa, Oliver Pybus, Moritz U.G. Kraemer, Eduan Wilkinson, Joana Isidro, Miguel Pinto, João Paulo Gomes, Cheryl Baxter, Richard Lessells, Ahmed E Ogwell, Yenew Kebede, Houriiyah Tegally, Áine O’Toole, Nikita Sitharam, Trevor Bedford, Sofonias K. Tessema, Tulio de Oliveira, Vitor Borges, Richard A. Neher, Andrew Rambaut

The continued detection of monkeypox cases around the world has led to a similar increase in the sequencing of these cases, and thus the availability of increasing genetic information about the current cases. As suggested in Happi et al. (2022) [[1](#ref_1)], we have extended the proposed Pango-based nomenclature scheme to designate additional lineages within the B.1 ‘outbreak’ clade. The continued designation of new lineages of the B.1 outbreak is particularly timely in order to help avoid geographical or otherwise stigmatizing names (for example, presumed associations with particular events or locations) while providing useful handles for the scientists, clinicians, and healthcare workers who need to discuss specific groups on the phylogenetic tree. 

We propose the initial designation of **five new lineages stemming from the B.1 lineage: B.1.1 - B.1.5** [[2](#ref_2),[3](#ref_3)]. The criteria used to designate these lineages was based on genomes available on INSDC databases and:
- Having at least 1 mutation above the B.1 polytomy
- Containing at least 14 sequences
- Clear common phylogenetic structure (no uncertainty about possibly being designated as 2 lineages instead of 1)

Additionally, we have begun to provide “reference sequences” for each clade/lineage in the monkeypox phylogeny, including the newly designated lineages [[4](#ref_4)]. These sequences were chosen based on their availability on INSDC databases and:
- Completeness (minimizing terminal and internal Ns and ambiguous nucleotides)
- Basality (if no single basal sequence is available, multiple are chosen so the MRCA is basal)
- Good quality (not showing unusually large numbers of frame shifts and/or stop codons)

It should be especially noted that the designation of a lineage does not imply any biological or phenotypic difference; indeed, some lineages are designated based only on synonymous mutations. Additionally, lineages can only be designated based upon the information available, which likely does not equitably represent the true geographical spread of monkeypox cases, and may also not fully reflect the genetic diversity of the virus. Currently, the proportion of sequenced cases, and their availability on INSDC databases, differs greatly between countries. Thus, lineages may be disproportionately associated with countries that sequence the most, such as Germany and Canada, and may not fully reflect the true geographic distribution of genetically similar cases. **Lineages and the sequences they contain should not be used alone to draw epidemiological conclusions.**

As the number of monkeypox sequences continues to increase, we will continue to monitor and designate additional lineages, including outside of the B.1 clade, as appropriate. 

_References:_
<a name="ref_1"></a>
1. Happi et al., Virological.org, Jun 2022 ([link](https://virological.org/t/urgent-need-for-a-non-discriminatory-and-non-stigmatizing-nomenclature-for-monkeypox-virus/853))
<a name="ref_2"></a>
2. Monkeypox Nextstrain build highlighting the new lineages: [link](https://nextstrain.org/monkeypox/hmpxv1?f_clade_membership=B.1.1,B.1.2,B.1.3,B.1.4,B.1.5&label=clade:B.1)
<a name="ref_3"></a>
3. Github Pull Request designating the new lineages: [link](https://github.com/nextstrain/monkeypox/pull/95)
<a name="ref_4"></a>
4. File with the reference sequences: **add link when approved**
