# Designation of three B.1 sublineages


Continued sequencing and sharing of monkeypox genomes has further expanded the diversity of the B.1 ('outbreak') clade, leading to the designation of **three further B.1 lineages: B.1.6 - B.1.8**.

The criteria used to designate these lineages is the same as previously, and is based on genomes available in INSDC databases and:
- International spread
- Having at least 1 mutation above the B.1 polytomy
- Containing at least 15 sequences or plausibly represents undersampled diversity
- Clear common phylogenetic structure (no uncertainty about possibly being designated as 2 lineages instead of 1)
- Has at least one freely available high quality reference sequence

Notably, lineage B.1.6 has not been detected outside of Peru. However, it has a unique mutation that is not seen elsewhere, and seems to be highly dominant in Peru. Thus, designation seems to be in the public health interest.

#### B.1.6

At date of designation, this is a unique Peruvian lineage defined by nucleotide `A` at position `111029`. At date of designation, the cluster is comprised of around 70 sequences (not all available on INSDC databases; see GISAID for additional sequences).
This lineage was an external proposal in [PR 9](https://github.com/mpxv-lineages/lineage-designation/issues/9) by [@Vjimenez-vasquez](https://github.com/Vjimenez-vasquez).
The `yaml` file can be found [here](../lineages/B.1.6.yml).

#### B.1.7

At date of designation, this lineage was circulating in a variety of countries including Germany, the UK, the USA, Portugal, South Africa and Mexico, and contains around 50 sequences. It is defined by nucleotide `T` and position `25644`.
This lineage was internally proposed in [PR 10](https://github.com/mpxv-lineages/lineage-designation/issues/10).
The `yaml` file can be found [here](../lineages/B.1.7.yml).

#### B.1.8

At date of designation, this lineage is circulating in Germany, Spain, and the USA, and contains around 20 sequences. It is defined by nucleotide `A` at position `5595` and `T` at position `191615`. Notably, these mutations are in the inverted terminal repeat, so one corresponds to the other (they are not independent). In phylogenetic representations where the inverted repeat is masked (such as the Nextstrain tree), only one mutation will be visible.
This lineage was internally proposed in [PR 11](https://github.com/mpxv-lineages/lineage-designation/issues/11).
The `yaml` file can be found [here](../lineages/B.1.8.yml).